.circleci/config.yml

```diff
@@ -281,6 +287,7 @@ jobs:
             - run: pip install --upgrade pip
             - run: pip install .[sklearn,tf-cpu,testing,sentencepiece,tf-speech,vision]
             - run: pip install tensorflow_probability
+            - run: pip install https://github.com/kpu/kenlm/archive/master.zip
```

Comment:
Do we need it in the TF tests?

Reply:
It's a `processor` and therefore framework independent - it's written in pure Python.

Reply:
So I kinda see it as a tokenizer and therefore should pretty much be installed everywhere where `sentencepiece` is installed as well

Reply:
What do you think?

Reply:
What do you think?